### PR TITLE
Update totalconnect.markdown

### PR DESCRIPTION
**Pull request by austinmroczek**
**Description:**
Update docs to show new configuration entry requirement.  

Also adds suggestion to use a separate, non administrative account Total Connect account with HA.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** https://github.com/home-assistant/home-assistant/pull/24427

## Checklist:

- [next] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9589"></a>

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please rebase to the `next` branch

**Review by frenck — CHANGES_REQUESTED** [review]
@austinmroczek this PR targets the wrong branch. It currently targets `current`, but it should target the `next` branch. Please change the target and rebase if needed.

**Comment by austinmroczek** [comment]
Sorry I’ve been out of town. Hope to get to it this weekend

**Comment by austinmroczek** [comment]
Changed to Next.  Thanks

File: source/_components/totalconnect.markdown [hunk 1]
```diff
@@ -33,6 +33,13 @@ alarm_control_panel:
     password: YOUR_PASSWORD
 ```
 
+Starting in Home Assistant version 0.95, add the following lines to your `configuration.yaml`:
```
Comment (frenck) [inline source/_components/totalconnect.markdown]:
```suggestion
Add the following lines to your `configuration.yaml`:

```

File: source/_components/totalconnect.markdown [hunk 2]
```diff
@@ -33,6 +33,13 @@ alarm_control_panel:
     password: YOUR_PASSWORD
```
Comment (frenck) [inline source/_components/totalconnect.markdown]: Please remove the previous instructions, they are no longer valid.

File: source/_components/totalconnect.markdown [hunk 3]
```diff
@@ -48,6 +55,8 @@ password:
   type: string
 {% endconfiguration %}
 
+You are highly encouraged to create a Total Connect user account specifically for Home Assistant.  It should not have full administrative privileges.
```
Comment (frenck) [inline source/_components/totalconnect.markdown]:
```suggestion
You are highly encouraged to create a Total Connect user account specifically for Home Assistant. It should not have full administrative privileges.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by austinmroczek** [comment]
Made updates per recommendations.  Thanks

**Review by frenck — DISMISSED** [review]
Thanks for the adjustments @austinmroczek! 👍 

✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by klaasnicolaas** [comment]
⚠️ We have I merge conflict

**Review by frenck — APPROVED** [review]

**Pull request merged**